polaris-react/src/components/Tooltip/components/TooltipOverlay/TooltipOverlay.scss

```
@@ -7,7 +7,8 @@
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   margin: var(--pc-tooltip-overlay-offset) var(--p-space-4) var(--p-space-4);
   opacity: 1;
-  box-shadow: var(--p-shadow-popover);
+  // stylelint-disable-next-line -- hardcoding an extra shadow for the Tooltip to enable it to be more visible against white backgrounds
+  box-shadow: var(--p-shadow-popover), 0 0 20px 8px rgba(23, 24, 24, 0.02);
```

```suggestion
  box-shadow: var(--p-shadow-popover), var(--p-shadow-deep);
```
Can we do something like this? It seems to only increase in contrast which should be a good thing and we stick with tokens

---

Also wanted to add quickly that in v11 we will be switching from our current [`depth` tokens](https://github.com/Shopify/polaris/blob/main/polaris-tokens/src/token-groups/depth.ts) to these new [`shadow` tokens](https://github.com/Shopify/polaris/blob/main/polaris-tokens/src/token-groups/shadow.ts) (we have [a migration](https://github.com/Shopify/polaris/pull/8272/files#diff-39620b2a876df203007dbee19f2d949f335ec9403499f63a3c0eb1dfbb2afeb5R7-R18) that maps all current values to new ones but if you want to be able to hand pick the values yourself I'd recommend using the new shadow tokens here instead).

---

Looking at the upcoming shadow tokens, it seems that `--p-shadow-lg` will do the trick

---

Looks like `--p-shadow-lg` doesn't exist currently, are we okay to hardcode it for now with a comment to update it once we move to v11.0?

---

It should exist. You might need to run a local build and get latest from `main` if it isn't there. https://github.com/Shopify/polaris/blob/764f17d86c30c9f30bec6717c09f07b2e04e23f8/polaris-tokens/src/token-groups/shadow.ts#L46-L49